insert.sql

```diff
@@ -0,0 +1,90 @@
+insert into `Picture`(
```

Comment:
Please use UPPERCASE for all SQL specific key words such as INSERT, CREATE, TABLE, FROM, SELECT etc